src/service.rs

```diff
@@ -630,13 +624,13 @@ where
     /// Runs the receiving key sampling protocol on the signer.
     #[inline]
     pub async fn receiving_keys(self, request: ReceivingKeyRequest) -> Result<Vec<ReceivingKey>> {
-        info!("[REQUEST] processing `receivingKeys`: {:?}", request)?;
-        let response = self.state.lock().signer[request.network].receiving_keys(request.message);
+        // info!("[REQUEST] processing `receivingKeys`: {:?}", request)?;
+        let response = self.state.lock().signer[request.network].address();
         info!(
             "[RESPONSE] responding to `receivingKeys` with: {:?}",
             response
         )?;
-        Ok(response)
+        Ok(vec![response])
```

Comment:
Do we need return Vec<Address> or just Address.

PS: `ReceivingKey` here just alia name for `Address`

Reply:
We have 1 address per account now so probably just 1 address

Reply:
Ok, I'll just return Address.